include/leveldb/cache.h

```diff
@@ -31,7 +31,7 @@ extern DLLX Cache* NewLRUCache(size_t capacity);
 
 class DLLX Cache {
  public:
-  Cache() { }
+  Cache():rep_(nullptr) { }
```

Comment:
This is very nitpicky but . . . spacing
```suggestion
  Cache() : rep_(nullptr) { }
```

Reply:
@aawoodwa Thank you, I've revised it.